examples/notebooks/healthcare/beam_nlp.ipynb

```diff
@@ -0,0 +1,968 @@
+            "Requirement already satisfied: googleapis-common-protos<2.0.dev0,>=1.56.2 in /usr/local/lib/python3.10/dist-packages (from google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,<3.0.0dev,>=1.32.0->google-cloud-aiplatform<2.0,>=1.26.0->apache-beam[gcp]) (1.59.1)\n",
+            "Requirement already satisfied: google-crc32c<2.0dev,>=1.0 in /usr/local/lib/python3.10/dist-packages (from google-resumable-media<3.0dev,>=0.6.0->google-cloud-bigquery<4,>=2.0.0->apache-beam[gcp]) (1.5.0)\n",
+            "Requirement already satisfied: pyasn1>=0.1.7 in /usr/local/lib/python3.10/dist-packages (from oauth2client>=1.4.12->google-apitools<0.5.32,>=0.5.31->apache-beam[gcp]) (0.5.0)\n",
+            "Requirement already satisfied: h11<0.15,>=0.13 in /usr/local/lib/python3.10/dist-packages (from httpcore>=0.17.3->dnspython<3.0.0,>=1.16.0->pymongo<5.0.0,>=3.8.0->apache-beam[gcp]) (0.14.0)\n",
+            "Requirement already satisfied: anyio<5.0,>=3.0 in /usr/local/lib/python3.10/dist-packages (from httpcore>=0.17.3->dnspython<3.0.0,>=1.16.0->pymongo<5.0.0,>=3.8.0->apache-beam[gcp]) (3.7.1)\n",
+            "Requirement already satisfied: exceptiongroup in /usr/local/lib/python3.10/dist-packages (from anyio<5.0,>=3.0->httpcore>=0.17.3->dnspython<3.0.0,>=1.16.0->pymongo<5.0.0,>=3.8.0->apache-beam[gcp]) (1.1.2)\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Note that below, **us-central1** is hardcoded as the location. This is because of the limited number of [locations](https://cloud.google.com/healthcare-api/docs/how-tos/nlp) the API currently supports."
+      ],
+      "metadata": {
+        "id": "D7lJqW2PRFcN"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "#Change this variable to True if you want to debug the Interactive Runner Pipeline else it uses Dataflow\n",
+        "debug = False\n",
+        "DATASET=\"<YOUR_DATASET_NAME>\"\n",
+        "TEMP_LOCATION=\"<YOUR_TEMP_LOCATION>\"\n",
+        "PROJECT='<YOUR_PROJECT_ID>'\n",
+        "LOCATION='us-central1'\n",
+        "URL=f'https://healthcare.googleapis.com/v1/projects/{PROJECT}/locations/{LOCATION}/services/nlp:analyzeEntities'\n",
+        "NLP_SERVICE=f'projects/{PROJECT}/locations/{LOCATION}/services/nlp'\n",
+        "GCS_BUCKET=PROJECT"
+      ],
+      "metadata": {
+        "id": "s9lhe5CZ5F3o"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "**BigQuery Setup**\n",
+        "\n",
+        "We will be using BigQuery to warehouse the structured data revealed in the output of the Healthcare NLP API. For this purpose, we create 3 tables to organize the data"
+      ],
+      "metadata": {
+        "id": "DI_Qkyn75LO-"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from google.cloud import bigquery\n",
```

Comment:
This is a pretty big block/hard to read - maybe we could split it into 3 cells (one for each table)?